### PR TITLE
Add stdout/stderr streaming for Dart debuggers.

### PR DESCRIPTION
**Pull request by turnidge**

### CODE REVIEW

File: sky/engine/core/script/dart_init.cc [hunk 1]
```diff
@@ -194,6 +194,28 @@ Dart_Handle GetVMServiceAssetsArchiveCallback() {
       ::dart::observatory::observatory_assets_archive_len);
 }
 
+static const char* kStdoutStreamId = "Stdout";
+static const char* kStderrStreamId = "Stderr";
```
Comment (abarth) [inline sky/engine/core/script/dart_init.cc]: `*  ->  []`

This is just a nit.  It makes the static data slightly better in the binary.

**Comment by abarth** [comment]
LGTM

File: sky/engine/core/script/dart_init.cc (hunk 1 above)
Comment (turnidge) [inline sky/engine/core/script/dart_init.cc]: Done.

**Comment by turnidge** [comment]
Ok, updated dart_init.cc to use [] instead of *.  PTAL.

**Comment by abarth** [comment]
LGTM

**Pull request merged**